.github/workflows/regular_ci.yml

```diff
@@ -82,13 +82,13 @@ jobs:
       fail-fast: false # Upload src/igr-tests/*/output/* files in igr-tests
       matrix:
         include:
-          - arch: 'amd64'
+          - arch: 'arm64'
```

Comment:
Is there even any point listing the architecture in a matrix, if there's only one and we don't control it?

Reply:
Hmm, Actually I did this to make the CI list look neat (and that's should be `aarch64` not `arm64` for this purpose).

Reply:
Do you think it should go?

Reply:
Is there any guarantee that it runs on a particular architecture? Is it documented anywhere?

Reply:
No, Most of them are so implicit. Also GitHub news is not a good source for checking these because it wasn't say that macos-latest will run on arm64. Also there is README for x86_64 builds of macos in [here](https://github.com/actions/runner-images/tree/main/images/macos) but looks like they aren't available in GitHub hosted runner because [The Document](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#standard-github-hosted-runners-for-public-repositories) says: M1 is used for MacOS-14 workflows.

Reply:
Oh so it is documented to run on M1 (that's ARM). Ok, put it as `aarch64` then.